notification/app/notification/controllers/Main.scala

```diff
@@ -34,7 +34,7 @@ final class Main(
 
   override def isPermittedTopic(apiKey: String): Topic => Boolean = {
     if (configuration.electionRestrictedApiKeys.contains(apiKey)) {
-      topic => topic.`type` == ElectionResults
+      topic => List(ElectionResults, LiveNotification).contains(topic.`type`)
```

Comment:
Yes, we need to rename electionRestrictedApiKeys to mobileInnovationLabRestrictedApiKeys.  It is so that the team in the US cannot accidentally send breaking news or other notifications that they don't need to.